### ISSUE TITLE
nuxtent maintainers / contributors

### ISSUE THREAD

**Issue by alidcastano** [nuxt-community/nuxtent-module#74]
I believe Nuxtent provides a great solution for devs who want to use Nuxt (and not another framework / static site generator) to build content heavy sites. So I'd love to see this project become more stable, but don't have as much time as I'd like to make it happen on my own. 

@medfreeman  @suhogrozdje  Would you two like to become Nuxtent maintainers/contributors? (You've both provided useful PRs, which is the reason I'm asking.)

**Comment by medfreeman** [comment]
@alidcastano sure, i share your point of view, and i'd be happy to !

**Comment by suhogrozdje** [comment]
Yes, I'd also be happy to, and I do have some free time here and now. Keep in mind that the pull request I made (a very small one, though) was my first one, so I am not very active on Github yet.

**Comment by alidcastano** [comment]
Great, invites sent. It'll give you read/write access to repo. :) 

@medfreeman Thanks for setting up e2e tests btw. I foolishly created Nuxtent without tests thinking its codebase wouldn't get too complicated...haha. But it's important to set those up so that Nuxtent can more easily accept improvements/PRs 

@suhogrozdje Yeah, I saw (congrats on your first PR!), but I read your comments on the issue and it was a well thought out and articulated solution. That one line of code allowed me to close three issues. :)

**Comment by alidcastano** [comment]
Btw, do you guys have sites running on Nuxtent? Or are they in progress? Would love to add them to README.

**Comment by medfreeman** [comment]
They are in progress ! I'm planning to make a nuxt template based on nuxt + nuxtent + netlify-cms + custom components, with a medium post to showcase it.

**Comment by alidcastano** [comment]
@medfreeman @suhogrozdje I created a slack channel so that we can communicate easier if you guys need to ask me anything.  Here's the [invite link](https://join.slack.com/t/nuxtent/shared_invite/enQtMjQ1MzI4MzgxNzI5LTcwOTlkYTczNTMzMGU1YjYyYjlkNzNjNDc3MjQ3YTdmMjc0ZThkZjM1ZWY4ZjhiZjc1MDIwYjJmMjAxODlhMzU).

Nuxtent is a small project so it shouldn't be much work but it'd be awesome to gradually improve it over time. :) 

@medfreeman That sounds awesome. Looking forward to seeing the project! If you'd like me to review the medium article let me know.

Closing this issue now. 👍
